test/functional/connections_stepdown.test.js

```diff
@@ -25,7 +25,7 @@ function expectPoolWasNotCleared(initialCount) {
   return count => expect(count).to.equal(initialCount);
 }
 
-describe('Connections survive primary step down', function () {
+describe.skip('Connections survive primary step down', function () {
```

Comment:
Can we add an associated NODE ticket to unskip here?

Reply:
Added todo comment to https://jira.mongodb.org/browse/NODE-3819